packages/contracts/src/utils/constants.ts

```diff
@@ -19,6 +19,8 @@ const TESTRPC_PRIVATE_KEYS_STRINGS = [
 export const constants = {
     INVALID_OPCODE: 'invalid opcode',
     REVERT: 'revert',
+    ERC20_PROXY_ID: 1,
+    ERC721_PROXY_ID: 2,
```

Comment:
We should use the constants from `AssetProxyId` from `'@0xproject/types` here. Or use these constants there. DRY.